backend/src/main/java/com/zeros/backend/service/RouteService.java

```diff
@@ -0,0 +1,9 @@
+package com.zeros.backend.service;
+
+import com.google.maps.model.TravelMode;
+
+public interface RouteService {
+
```

Comment:
We can find a better name for this class .
Route service has no meaning

Reply:
any suggestions !!

Reply:
I like what you are did here by creating the contract 👍

Reply:
We need something like RouteCalculator or estimator .

Reply:
@NaitYoussef @jyfar  any suggestions !

Reply:
RouteEsimator looks good to me